protocol/requests/create_account.go

```diff
@@ -59,10 +59,7 @@ type CreateAccount struct {
 
 	PreviewPrivacy bool `json:"previewPrivacy"`
 
-	VerifyTransactionURL     *string `json:"verifyTransactionURL"`
-	VerifyENSURL             *string `json:"verifyENSURL"`
```

Comment:
Was removing the ens one intended?

Reply:
Yes. It's just also never set in desktop, and defaults to mainnet in `status-go`. So I've removed it from here